app/terminal/terminal.tsx

```diff
@@ -1,9 +1,12 @@
 import React from "react";
 import { LazyLog } from "react-lazylog";
+import { eventlog } from "../../proto/eventlog_ts_proto";
+import rpcService from "../service/rpc_service";
 
 export interface TerminalProps {
   value?: string;
   lightTheme?: boolean;
+  invocationId?: string;
```

Comment:
It might give us more flexibility to keep this component agnostic to invocations. Maybe instead pass in a `fullLogFetcher`?

```
fullLogFetcher?: () => Promise<string>;
```